karma.conf.js

```diff
@@ -26,7 +26,7 @@ module.exports = function(config) {
 
 		// list of files / patterns to load in the browser
 		files: [
-			'http://cdn.polyfill.io/v2/polyfill.js?features=fetch&flags=gated',
+			'https://cdn.polyfill.io/v2/polyfill.js?flags=gated&ua=safari/4&features=default',
```

Comment:
Do we need fetch, it is not in the default polyfill set --> https://polyfill.io/v2/docs/features/#fetch